packages/vue/src/examples/form/Form.vue

```diff
@@ -101,6 +101,12 @@
         error-message="Please choose your country."
         @blur="emailBlur = false"
       />
+      <SfHeading title="Prices" level="5" />
+      <SfRange
+        v-model="range"
```

Comment:
I think there was a change in the naming of the props/variables. Type and value of `range`  looks similar to `value` in SfRange. Is it the same? Also I don't think it is being used in SfRange

Reply:
Here I'm passing array of values for setting sliders in SfRange. I think it's okay, but maybe I'm wrong.